lms/envs/common.py

```diff
@@ -2806,6 +2806,8 @@ def _make_locale_paths(settings):  # pylint: disable=missing-docstring
     "gender",
     "year_of_birth",
     "level_of_education",
+    "specialty",
+    "profession"
```

Comment:
Why are they being added here?

Reply:
> Why are they being added here?
After the change above these fields were rendered at the end of the form ever after terms and conditions so I have moved them up the list to maintain order.